src/rdf4cpp/rdf/parser/RDFFileParser.cpp

```diff
@@ -0,0 +1,48 @@
+#include "RDFFileParser.hpp"
+
+rdf4cpp::rdf::parser::RDFFileParser::RDFFileParser(const std::string &file_path, rdf4cpp::rdf::parser::ParsingFlags flags,
```

Comment:
Please wrap the definitions in `namespace rdf4cpp::rdf::parser { }`. Without it the signal-to-noise ratio is quite high.

Reply:
I see you wrapped it in `namespace` but did not shorten the identifiers. If you didn't know yet (nothing to worry about) if you wrap it in namespace (like you now did) you can shorten for example this:
```c++
rdf4cpp::rdf::parser::RDFFileParser::RDFFileParser(const std::string &file_path, rdf4cpp::rdf::parser::ParsingFlags flags,
                                                   rdf4cpp::rdf::storage::node::NodeStorage node_storage) {
    ...                                                  
}
```

like so:
```c++
namespace rdf4cpp::rdf::parser {

RDFFileParser::RDFFileParser(const std::string &file_path, ParsingFlags flags, 
                             storage::node::NodeStorage node_storage) {
    ...
}

...

}  // namespace rdf4cpp::rdf::parser
``` 

which is much more readable if there are many definitions. Thats why I originally suggested the change